proto/v1/event.proto

```diff
@@ -67,9 +67,14 @@ message EventCollectionRequest {
   // - discoveryDate
   // - occurrenceTime
   string date = 3;
+  // The property used for filtering
+  string filter_prop = 4;
+  // Event is not returned if value below the limit
+  double filter_limit = 5;
```

Comment:
Will this API only handle filtering by a lower limit? Currently in the UI, user can also choose an upper limit..

Reply:
My original intention was to use this reduce the size of the data. For a more comprehensive filtering, maybe we should start a doc and decide from that?

Reply:
could we have two optional limits (upper_limit, lower_limit)? if neither is set, all data is returned.

Reply:
Done